src/components/Header/index.jsx

```diff
@@ -68,33 +68,20 @@ const GordonHeader = ({ onDrawerToggle }) => {
   }, [width]);
 
   const createDialogBox = () => {
-    if (dialog === 'offline') {
-      return (
-        <GordonDialogBox
-          open={dialog}
-          onClose={() => setDialog(null)}
-          title={'Offline Mode'}
-          buttonClicked={() => setDialog(null)}
-          buttonName={'Okay'}
-        >
-          This feature is unavailable offline. Please reconnect to internet to access this feature.
-        </GordonDialogBox>
-      );
-    } else if (dialog === 'unauthorized') {
-      return (
-        <GordonDialogBox
-          open={dialog}
-          onClose={() => setDialog(null)}
-          title={'Credentials Needed'}
-          buttonClicked={() => setDialog(null)}
-          buttonName={'Okay'}
-        >
-          This feature is unavailable while not logged in. Please log in to access it.
-        </GordonDialogBox>
-      );
-    } else {
-      return null;
-    }
+    const isOffline = dialog === 'offline';
+    return (
+      <GordonDialogBox
+        open={dialog}
+        onClose={() => setDialog(null)}
+        title={isOffline ? 'Unavailabile Offline' : 'Login Required'}
+        buttonClicked={() => setDialog(null)}
+        buttonName={'Okay'}
+      >
+        {isOffline
+          ? 'That page is not available offline. Please reconnect to internet to access this feature.'
+          : 'That page is only available to authenticated users. Please log in to access it.'}
```

Comment:
I actually think that "that" is more appropriate here because these dialogs are being launched in response to the user clicking on a navigation link, without navigating them. For example, if I am on the homepage and click the `People` tab but I'm unauthenticated, I stay on the home page and get a dialog telling me "That page isn't available". "This page" wouldn't make sense in context because "this" means the current page, which is available.

Reply:
Huh, I guess that makes sense (no pun intended). Why are we showing the tab as an option? Is that just something we haven't considered that would take some work to implement?

Reply:
I believe it was an intentional choice to keep them visible but greyed out and disabled, because previously they did just disappear. I don't know the reasoning for that choice though. Possibly so that unauthenticated users could see some of the available features (i.e. a marketing thing from CC), or possibly so that users who wanted to access those pages would try to click them and get this message, instead of just not seeing them and not realizing they aren't signed in and so being confused.